app/Controllers/userController.php

```diff
@@ -36,12 +36,23 @@ public static function deleteFeverKey($username) {
 		return false;
 	}
 
-	public static function updateUser($user, $passwordPlain, $apiPasswordPlain, $userConfigUpdated = array()) {
+	public static function updateUser($user, $email, $passwordPlain, $apiPasswordPlain, $userConfigUpdated = array()) {
 		$userConfig = get_user_configuration($user);
 		if ($userConfig === null) {
 			return false;
 		}
 
+		if ($email !== null && $userConfig->mail_login !== $email) {
```

Comment:
Minor detail: Probably `$email != ''` would be safer here

Reply:
That was on purpose since the email should not be required when `force_email_validation` is `false`. But I realize that I should enforce the value if it's `true`.